notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -2932,6 +2932,51 @@ biological computer [Dr. Yuhara explained that Mechagodzilla was installed with
 nuclear weapons [Sara stated that it was the use of the H-bomb the first attracted Godzilla to Japan.]
 
 
+movie: Dreamcatcher (2003)
+==========================
+
+:: Title
+Dreamcatcher
+
+:: Description
+Dreamcatcher is a 2003 American science fiction horror film based on Stephen
+King's 2001 novel of the same name.
+
+Synopsis: Four friends encounter an invasion of parasitic aliens.
+
+:: Date
+2003-03-21
+
+:: References
+https://en.wikipedia.org/wiki/Dreamcatcher_(2003_film)
+
+:: Ratings
+3 <paul>
+
+:: Major Themes
+friendship [The story follows four male friends as they increasingly find themselves at the center of an invasion of parasitic aliens while otherwise minding their own business on their annual hunting trip in Maine.]
+telepathic ability [The four friends had all acquired telepathic powers after having saved a mentally disabled boy from bullies back when they were children. Their using of this special power was explored throughout the story.]
+mind reading ability [The four friends had all acquired the power to read minds after having saved a mentally disabled boy from bullies back when they were children. Their using of this special power was explored throughout the story.]
+extraterrestrial being [The large humanoid alien Mr. Gray was trying to contaminate the Boston water supply with parasitic alien larvae.]
+attack from outer space [Maine was the site of an invasion of parasitic aliens.]
+human vs. monster [Horrible alien worm creatures were going are parasitizing people.]
+monster [Horrible alien worm creatures were going are parasitizing people. In the end, the surviving friends faced off against a large alien worm creature.]
+covert military operation [Colonel Curtis was in charge of a secret military operation to exterminate the invading extraterrestrials.]
```

Comment:
Like with the "government secrecy" I mentioned in another review, I wonder if the "covertness" was really major here..?

Reply:
Maybe we need a theme for just military operation?

Reply:
Hum... they made a pretty big deal about there being an elite military unit specializing in extraterrestrials.

But in general, I bet a **military operation** theme would come in handy.